### PR TITLE
mods can't move mods to the listeners section

### PR DESCRIPTION
**Pull request by nadirabbas**
@benawad you said you wanna review backend changes

### CODE REVIEW

**Comment by benawad** [comment]
can you write a test case for this, let me know if its hard to figure out and I'll make a video on it

**Comment by nadirabbas** [comment]
@benawad Figuring it out shouldn't be a problem, but you should make a video on it nonetheless if you have time. To help all people contributing 🙏

**Review by A-Yamout — APPROVED** [review]

File: kibbeh/src/modules/room/UserPreviewModal.tsx [hunk 1]
```diff
@@ -71,7 +71,11 @@ const UserPreview: React.FC<{
     return <div className={`flex text-primary-100`}>This user is gone.</div>;
   }
 
-  const canDoModStuffOnThisUser = !isMe && (iAmCreator || iAmMod) && !isCreator;
```
Comment (BronzW) [inline kibbeh/src/modules/room/UserPreviewModal.tsx:82]: Can we change name on this variable to something more prof=

**Review by BronzW — APPROVED** [review]

File: kibbeh/src/modules/room/UserPreviewModal.tsx (hunk 1 above)
Comment (A-Yamout) [reply kibbeh/src/modules/room/UserPreviewModal.tsx:82]: On the discord server I asked and we decided on dogeperms

**Comment by amitojsingh366** [comment]
@nadirabbas what's the status of this?

**Review by benawad — CHANGES_REQUESTED** [review]
.

File: kousa/test/beef/user_test.exs [hunk 2]
```diff
@@ -64,7 +64,39 @@ defmodule Kousa.Beef.UserTest do
...
+               Users.edit_profile(id, %{
+                 username: "timmy",
+                 displayName: "tim",
+                 bio: "",
+                 avatarUrl:
+                   "https://cdn.discordapp.com/avatars/473965680857972757/6b3e4b9be1fd453230172ca6509f0b46.webp"
+               })
+    end
+
+    test "with avatar_url that is not from twitter/github/discord", %{user: %{id: id}} do
```
Comment (amitojsingh366) [inline kousa/test/beef/user_test.exs:99]: @benawad this was added by me right? in nadir's other pr

Comment (amitojsingh366) [reply kousa/test/beef/user_test.exs:99]: why adding it here also?

**Comment by nadirabbas** [comment]
@amitojsingh366 I had to merge remote's staging.

**Pull request closed without merging**